### ISSUE TITLE
Remove "Variable" from torch.cat error message

### ISSUE THREAD

**Issue by zou3519** [pytorch/pytorch#8694]
```
>>> import torch
>>> import numpy as np
>>> torch.cat((np.ones((5,5)), np.zeros((5,5))),1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: expected Variable as element 0 in argument 0, but got tuple
```

initially reported from https://discuss.pytorch.org/t/bad-error-message/19828

**Comment by Naman-ntc** [comment]
Also it says that `got tuple instead`, torch.cat takes tuple as argument!?
So that is even more confusing, it should maybe say something like `tuple elements should be tensors` or on that lines!

**Comment by vishwakftw** [comment]
This was fixed in #8309 .
